minwoo.lee_java/src/BOJ4386.java

```diff
@@ -0,0 +1,73 @@
+import java.util.*;
+import java.io.*;
+
+public class BOJ4386 {
+    static int N;
+    static Star[] stars;
+
+    public static void main(String[] args) throws IOException {
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        StringTokenizer st;
+        N = Integer.parseInt(br.readLine());
+        stars = new Star[N];
+        for (int i = 0; i < N; i++) {
+            st = new StringTokenizer(br.readLine());
+            double x = Double.parseDouble(st.nextToken());
+            double y = Double.parseDouble(st.nextToken());
+            Star star = new Star(x, y);
+            stars[i] = star;
```

Comment:
```suggestion
            stars[i] = new Star(x, y);
```
이런 인스턴스 변수들은 하나로 합쳐도 될 것 같습니다.

Reply:
수정하였습니다.